packages/admin-panel-server/src/viz-builder/dashboardVisualisation/combineDashboardVisualisation.ts

```diff
@@ -46,6 +46,8 @@ export function combineDashboardVisualisation(
   };
   if (!dashboardItem.legacy) {
     visualisation.permissionGroup = (report as Report).permissionGroup;
+    visualisation.latestDataParameters =
+      'latestDataParameters' in report ? report.latestDataParameters : {};
```

Comment:
Should they always be defined? Since it defaults to `{}`?

Reply:
If it's a viz being created then it won't have this property yet

Reply:
In that case can we just rely on the DB to initaliase this value? And we don't need to pass anything in